### PR TITLE
 Updated External Lib Apache Poi build process

### LINKED ISSUE #614: Update maven based library process for apache poi plugins
Update the apache.commons, apache cli, etc. plugins with the new maven absed process from #611.

### PR DESCRIPTION
**Pull request by ghost**
Updated Apache poi plugin with maven based dependency download #613 needs to be merged before.

Closes #614

### CODE REVIEW

File: de.dlr.sc.virsat.external.lib.apache.poi/build.properties [hunk 1]
```diff
@@ -1,8 +1,16 @@
 source.. = src/
 output.. = target/classes/
 bin.includes = META-INF/,\
-               .,\
                about.html,\
                about_files/,\
-               externalLib/
+               externalLib/,\
+               pom.xml,\
+               externalLib/maven/commons-codec.jar,\
```
Comment (andriikovalov-dlr) [inline de.dlr.sc.virsat.external.lib.apache.poi/build.properties:8]: We probably don't need to explicitly list all jars

File: de.dlr.sc.virsat.javadoc.api/help/api_docs.xml [hunk 2]
```diff
@@ -2,265 +2,8 @@
 <?NLS TYPE="org.eclipse.help.toc"?><toc label="VirSat Core API JavaDoc Compilation">
 <topic href="help/api-docs/javadoc/overview-summary.html" label="VirSat Core API JavaDoc Compilation">
 <topic label="Packages">
-<topic href="help/api-docs/javadoc/com/github/cliftonlabs/json_simple/package-summary.html" label="com.github.cliftonlabs.json_simple"/>
```
Comment (andriikovalov-dlr) [inline de.dlr.sc.virsat.javadoc.api/help/api_docs.xml:5]: Suspicious changes

**Review by andriikovalov-dlr — APPROVED** [review]
Looks good apart from a couple of minor things.

File: de.dlr.sc.virsat.external.lib.apache.poi/build.properties (hunk 1 above)
Comment (PhilMFischer) [reply de.dlr.sc.virsat.external.lib.apache.poi/build.properties:8]: Correct, we can try to just sue the folder once we update the versions

File: de.dlr.sc.virsat.javadoc.api/help/api_docs.xml (hunk 2 above)
Comment (PhilMFischer) [reply de.dlr.sc.virsat.javadoc.api/help/api_docs.xml:5]: Not an issue, the api_docs.xml gets regenerated by the maven build process when creating the java docs

**Review by SaMuellerDLR — APPROVED** [review]
Approved!

**Pull request merged**